src/proteus/dswx_hls.py

```diff
@@ -4377,8 +4444,10 @@ class in the browse image. False to display PSW-Agg.
                           flag_collapse_wtr_classes=FLAG_COLLAPSE_WTR_CLASSES,
                           output_files_list=build_vrt_list)
 
-    cloud, wtr_layer = _compute_and_apply_cloud_layer(wtr_2_layer, fmask,
+    cloud, wtr_layer = _compute_and_apply_cloud_layer(
+        wtr_2_layer, preliminary_cloud_layer, fmask,
         mask_adjacent_to_cloud_mode)
+    del preliminary_cloud_layer
```

Comment:
Please see previous comment about how to better modularize the code.

Reply:
> Hi @gshiroma , Thank you for updating the version number and adding the new metadata.
> 
> As you'll see in the comments, I'm a bit concerned about a couple of the new metrics, and was quite confused by splitting the computation of the cloud layer into two parts. The first might require a larger conversation with the group, and the second I provided a suggestion for how to streamline the code a bit better.
> 
> After an initial look at the outputs/metadata, everything seems ok, but I'll take a deeper look after these changes are implemented in case they introduce new bugs. Thanks!

Thank you so much for your quick review, @nemo794 ! I answered/addressed all your comments. The only remaining comments are related to the evergreen class 201 that I forward the question to John. Please, let me know if you have any other comments or concerns.